packages/bootstrap/scss/core/typography/index.import.scss

```diff
@@ -0,0 +1,100 @@
+
+/// The base font size across all components.
+/// @group typography
+$kendo-font-size: $font-size-base !default;
+/// The extra small font size across all components.
```

Comment:
$kendo-font-size-xxs is missing here -> is this by purpose?

Reply:
It is omitted on purpose and I am questioning whether to remove it altogethera as we are not using it anywhere

Reply:
I would suggest to keep this value -> it is 0.5rem and is possible to use it in the future :)